

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m16:00:59  Running with dbt=1.7.17
[0m16:00:59  Registered adapter: firebolt=1.6.1
[0m16:00:59  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 418 macros, 0 groups, 0 semantic models
[0m16:00:59  
[0m16:01:00  Concurrency: 1 threads (target='default')
[0m16:01:00  
[0m16:01:00  1 of 1 START seed file seed .................................................... [RUN]
[0m16:01:02  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 1.28s]
[0m16:01:02  
[0m16:01:02  Finished running 1 seed in 0 hours 0 minutes and 2.25 seconds (2.25s).
[0m16:01:02  
[0m16:01:02  [32mCompleted successfully[0m
[0m16:01:02  
[0m16:01:02  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m16:01:02  Running with dbt=1.7.17
[0m16:01:02  Registered adapter: firebolt=1.6.1
[0m16:01:02  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 418 macros, 0 groups, 0 semantic models
[0m16:01:02  
[0m16:01:03  Concurrency: 1 threads (target='default')
[0m16:01:03  
[0m16:01:03  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m16:01:04  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 1.20s]
[0m16:01:04  
[0m16:01:04  Finished running 1 incremental model in 0 hours 0 minutes and 2.14 seconds (2.14s).
[0m16:01:04  
[0m16:01:04  [32mCompleted successfully[0m
[0m16:01:04  
[0m16:01:04  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m16:01:04  Running with dbt=1.7.17
[0m16:01:04  Registered adapter: firebolt=1.6.1
[0m16:01:04  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 418 macros, 0 groups, 0 semantic models
[0m16:01:04  
[0m16:01:05  Concurrency: 1 threads (target='default')
[0m16:01:05  
[0m16:01:05  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m16:01:06  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 1.12s]
[0m16:01:06  
[0m16:01:06  Finished running 1 incremental model in 0 hours 0 minutes and 2.05 seconds (2.05s).
[0m16:01:06  
[0m16:01:06  [31mCompleted with 1 error and 0 warnings:[0m
[0m16:01:06  
[0m16:01:06    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "a663e047-9d5d-414c-bcd6-66826d2f5040",
      "request_id": "be65b7ed-2dad-45d9-a7c5-1b1a7f27d0ed"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m16:01:06  
[0m16:01:06  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
